pkgs/applications/editors/emacs/default.nix

```diff
@@ -44,6 +44,7 @@ stdenv.mkDerivation rec {
   patches = [
     ./clean-env.patch
     ./tramp-detect-wrapped-gvfsd.patch
+    ./support-hunspell-1.7.0-in-ispell.el.patch  # should drop at next package update
```

Comment:
The patch is fetchable (eg. using fetchpatch) from here: https://git.savannah.gnu.org/cgit/emacs.git/patch/?id=2925ce5a7ec1424cfaea9f2f86bd3cab27832584

Reply:
Thanks, I wasn't aware of fetchpatch